src/lib/registration/url_helpers.rb

```diff
@@ -119,6 +120,13 @@ def self.reg_url_at_installation
       slp_service_url
     end
 
+    # get registration URL from AutoYaST configuration file
+    def self.reg_url_from_autoyast_config
+      server = ::Registration::Storage::Config.instance.reg_server
```

Comment:
Um, where is the `reg_server` value set during the AutoYaST installation? There is actually a [`set_registration_url` method defined in the scc_auto.rb client](https://github.com/yast/yast-registration/blob/5d614c6fb3787690ea377eb196e8c96d235f189d/src/clients/scc_auto.rb#L144) called at [`Write` here](https://github.com/yast/yast-registration/blob/5d614c6fb3787690ea377eb196e8c96d235f189d/src/clients/scc_auto.rb#L173).